azurerm/resource_arm_api_management_api_test.go

```diff
@@ -28,6 +28,8 @@ func TestAccAzureRMApiManagementApi_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "soap_pass_through", "false"),
 					resource.TestCheckResourceAttr(resourceName, "is_current", "true"),
 					resource.TestCheckResourceAttr(resourceName, "is_online", "false"),
+					resource.TestCheckResourceAttr(resourceName, "authentication_settings.#", "1"),
```

Comment:
This is set by default on creation of APIs for APIM, so I thought it was easiest to verify on the "basic" test. If you want an explicit test let me know.